torch/_inductor/cudagraph_trees.py

```diff
@@ -753,6 +753,8 @@ def __init__(
         self.device = device_index
         self.stack_traces = stack_traces
         self.stream = stream
+        # Used to only check this once
+        self.static_inputs_stable = False
```

Comment:
I don't think we need to update this from True/False. 

If CUDAGraphNode is instantiated with `torch._dynamo.config.inline_inbuilt_nn_modules = True`:

- in check invariants, check static data ptrs
- skip check_static_inputs_are_stable

If False:

- dont check static data ptrs in check invariants
- do check_static_inputs_are_stable

Also add comment about what it represents

Reply:
yeah got it, we can just burn this in